core/src/main/java/org/apache/iceberg/MetricsConfig.java

```diff
@@ -104,38 +108,40 @@ public static MetricsConfig forTable(Table table) {
    * @param table an Iceberg table
    */
   public static MetricsConfig forPositionDelete(Table table) {
-    MetricsConfig config = new MetricsConfig();
+    ImmutableMap.Builder<String, MetricsMode> columnModes = ImmutableMap.builder();
```

Comment:
can we use ImmutableMap.of here?

Reply:
+1, since there are only 2 values.

Reply:
I think this will need to be a builder because it adds additional column modes at L123, resolve the comment.